mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss

```diff
@@ -1,14 +1,24 @@
 @import "src/styles/variables";
 
 .dropdownMenuItem {
-  padding: $paddingSmall;
+  padding: $paddingSmall $paddingMedium;
+  font-size: $fontSizeMedium;
+  cursor: pointer;
 
-  &:hover {
-    background-color: var(--secondaryBgColor);
-    cursor: pointer;
+  &:not(:last-child) {
+    border-bottom: $primaryBorder solid var(--tertiaryBgColor);
```

Comment:
border is not background wrong token

Reply:
![image](https://github.com/tritonJS826/masters-way/assets/112805668/29836275-2ee1-4dab-a4df-34b4e211d1e5)

Reply:
fixed